chainer/testing/condition.py

```diff
@@ -1,6 +1,7 @@
 import functools
 import unittest
 
+import _pytest
```

Comment:
I observed
```
% python -c 'import _pytest; _pytest.outcomes'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module '_pytest' has no attribute 'outcomes'
%
```
and
```
% python -c 'import _pytest; import pytest; _pytest.outcomes'
%
```

Reply:
Thanks, fixed in 4a8a90ab87f088c571567f98b6d69e0d1d535aa0.